header.go

```diff
@@ -242,7 +242,7 @@ func preloadableHeaderMaxAge(hstsHeader HSTSHeader) Issues {
 		issues = issues.addWarningf(
 			"header.preloadable.max_age.over_10_years",
 			"Max-age > 10 years",
-			"FYI: The max-age (%d seconds) is longer than 10 years, which is an unusually long value.",
+			"FYI: The max-age (%d seconds) is longer than 10 years, which is an unusually long value, and may be disregarded by some browsers.",
```

Comment:
The wording "may be disregarded" makes it sound like the browser will disregard the max-age value (and thus the HSTS header) altogether, which is not the case. How about "may be capped at a lower value" instead?

Reply:
That's a valid concern. Revising.